NEWS

```diff
@@ -1,5 +1,7 @@
 # Machinery Release Notes
 
+* The x86_64 machinery-helper is now shipped with machinery. It improves the inspection
+  performance of unmanaged-files when the files are not extracted
```

Comment:
Maybe more simple "It speeds up inspection of unmanaged-files when the files are not extracted"?